### ISSUE TITLE
okteto up replicas should be 1 in services

### ISSUE THREAD

**Issue by jLopezbarb** [okteto/okteto#3485]
**Describe the bug**
We are not changing the number of replicas specified in the services section under the dev section ([we do for the main service](https://github.com/okteto/okteto/blob/master/pkg/k8s/apps/translate.go#L81)). We should always have one replica minimum in the services that are in detached dev mode.

**To Reproduce**

1. Deploy https://github.com/okteto/movies-with-helm
2. Run the following command: `kubectl scale deployment frontend --replicas=0` 
3. Modify the `okteto.yml` to be:
```yaml
icon: https://apps.okteto.com/movies/icon.png

build:
  api:
    context: api
  frontend:
    context: frontend

deploy:
  - helm upgrade --install movies chart
    --set api.image=${OKTETO_BUILD_API_IMAGE}
    --set frontend.image=${OKTETO_BUILD_FRONTEND_IMAGE}

dev:
  api:
    command: yarn start
    sync:
      - api:/src
    forward:
      - 9229:9229
    services:
    - selector:
         app.kubernetes.io/component: frontend
      sync:
      - frontend:/usr/src/app
  frontend:
    image: okteto/node:14
    command: bash
    sync:
      - frontend:/usr/src/app
```
3. Run `okteto up`
4. Wait until it's in development
5. Run  `kubectl get deployments` and check that the frontend replicas is 0

**Expected behavior**
It should have at least 1 replica. If the number of replicas is bigger than 1 we should keep that number for the services

**Desktop (please complete the following information):**

- Version 2.14.0

**Comment by jLopezbarb** [comment]
Related issue https://github.com/okteto/okteto/issues/3363

**Comment by AnshulAngaria** [comment]
@jLopezbarb I would like to work on this issue.

**Comment by jLopezbarb** [comment]
Sure, @AnshulAngaria. I assigned the issue to you. Feel free to reach out if you need more context on the issue or where all the transformations happens

**Comment by AnshulAngaria** [comment]
@jLopezbarb I have submitted a PR, can you please review.

**Comment by pchico83** [comment]
@AnshulAngaria could we modify the behavior?
By default, I would keep the `replicas` from the original deployment. But we should have a field `replicas`, only supported under `services`, to specify the number of replicas after `okteto up`.
It's importart than, when `okteto down` is executed, we restored the original replicas of the deployment specified in `services`

**Comment by jLopezbarb** [comment]
@pchico83 we have [this other issue](https://github.com/okteto/okteto/issues/3363) for what you are mentioning. Should we tackle it in this ticket or use the other ticket to fix it?

**Comment by AnshulAngaria** [comment]
@pchico83 @jLopezbarb I can add this behaviour in this PR. This will close the issue #3363 as well.

**Comment by pchico83** [comment]
@AnshulAngaria what ever works better for you, 1 or two PRs are fine. I think the logic is over then same place.
We should also create another issue to support `replicas`. This way, you could change the `replicas` of `services` in okteto up. This is useful in some scenarios where you want to enable a dev container and set `replicas: 0` in all the deployments referenced by `services`

**Comment by AnshulAngaria** [comment]
@pchico83 @jLopezbarb I made the following changes
1. Added a `replicas` field in the manifest to specify the required number of replicas for `services`.
2. If the replicas are not specified in `services` manifest and if the original deployment has 0 replicas, create one service replica.
3. Fixed the issue #3363 where okteto down was not scaling back original deployment replicas specified in `services`.
